internal/remediation/relax/relax.go

```diff
@@ -23,7 +23,7 @@ type RequirementRelaxer interface {
 	Relax(ctx context.Context, cl resolve.Client, req resolve.RequirementVersion, allowMajor bool) (resolve.RequirementVersion, bool)
 }
 
-func GetRelaxer(ecosystem resolve.System) (RequirementRelaxer, error) {
+func Get(ecosystem resolve.System) (RequirementRelaxer, error) {
```

Comment:
Now that it's called relax / suggest, we probably can keep it as GetRelaxer:
```
relaxer := relax.GetRelaxer(eco)
// vs
relaxer := relax.Get(eco)
```

Reply:
But I don't feel strongly about it either way, so up to you, happy to merge it as is.

Reply:
+1. I think `.Get()` would make sense when the package name is what is being gotten.

Reply:
done!